src/configure

```diff
@@ -1244,6 +1244,15 @@ elif [ "`uname`" == "Linux" ]; then
       echo MKLLIB = $MKLLIBDIR >> kaldi.mk
     fi
     echo >> kaldi.mk
+
+    echo "MKL_LD_PRELOAD := $MKLROOT/lib/intel64/libmkl_def.so" >> kaldi.mk
+    echo "MKL_LD_PRELOAD += $MKLROOT/lib/intel64/libmkl_avx2.so" >> kaldi.mk
```

Comment:
Hm, it seems to me it  would be more straightforward to do
"echo MKL_LD_PRELOAD := $MKLROOT/lib/intel64/libmkl_def.so:\
$MKLROOT/lib/intel64/libmkl_avx2.so:\
.... "  >> kaldi.mk
then no need for substitution magic.

Reply:
I also tried the suggested approach. The problem is that the resulting `configure`
file cannot be properly indented, e.g.,

```
"echo MKL_LD_PRELOAD := $MKLROOT/lib/intel64/libmkl_def.so:\
<no space here>$MKLROOT/lib/intel64/libmkl_avx2.so:\
<no space here>.... "  >> kaldi.mk
```

Reply:
In addition, if indentation is not the problem, the resulting `kaldi.mk` would have a very long line for `MKL_LD_PRELOAD` which harms readability.

Reply:
@danpovey @naxingyu 
It happens that `LD_PRELOAD` accepts both colon `:` and space-separated paths.

Please refer to http://man7.org/linux/man-pages/man8/ld.so.8.html

>LD_PRELOAD
>              A list of additional, user-specified, ELF shared objects to be
>             loaded before all others.  This feature can be used to selec‐
>             tively override functions in other shared objects.
>
>             The items of the list can be separated by spaces ****or**** colons,
>             and there is no support for escaping either separator.

Reply:
Should it be marked as resolved?